pywbemtools/pywbemcli/_cmd_statistics.py

```diff
@@ -0,0 +1,338 @@
+# (C) Copyright 2021 IBM Corp.
+# (C) Copyright 2021 Inova Development Inc.
+# All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Click command definition for the stastic command group which includes
+cmds for managing the gathering and display of statistics that pywbemcli
+and the WBEM server can gather on the WBEM operations
+
+NOTE: Commands are ordered in help display by their order in this file.
+"""
+
+from __future__ import absolute_import, print_function
+
+import click
+
+from pywbem import Error
+
+from .pywbemcli import cli
+from ._common import warning_msg, CMD_OPTS_TXT, GENERAL_OPTS_TXT, \
+    SUBCMD_HELP_TXT
+from ._common_options import add_options, help_option
+from ._click_extensions import PywbemcliGroup, PywbemcliCommand
+
+# NOTE: A number of the options use double-dash as the short form.  In those
+# cases, a third definition of the options without the double-dash defines
+# the corresponding option name, ex. 'include_qualifiers'. It should be
+# defined with underscore and not dash
+
+# Issue 224 - Exception in prompt-toolkit with python 2.7. Caused because
+# with prompt-toolkit 2 + the completer requires unicode and click_repl not
+# passing help as unicode in options as unicode
+# NOTE: Insure that all option help attributes are unicode to get around this
+#       issue
+
+
+@cli.group('statistics', cls=PywbemcliGroup, options_metavar=GENERAL_OPTS_TXT,
+           subcommand_metavar=SUBCMD_HELP_TXT)
+@add_options(help_option)
+def statistics_group():
+    """
+    Command group for WBEM server statistics.
+
+    This command group defines commands to enable, disable and control the
+    display of statistics that are displayed. Since stattistical information
+    is gathered both in the pywbem client and the server, there are controls
+    for both elements.
+
+    This command group allows enabling the gathering of statistics in a
+    WBEM server that include the statistics capability.
+
+    In addition to the command-specific options shown in this help text, the
+    general options (see 'pywbemcli --help') can also be specified before the
+    'server' keyword.
```

Comment:
After thinking about this some more, it seems to me that we should keep them separate, because it is not easy to relate the server statistics as implemented by OpenPegasus to WBEM operations, and because server statistics are not standardized, and that is handled better by keeping client and server statistics separate.

Reply:
We want keep these separate.

Reply:
I made no changes here.